lib/graphql_practice_web/resolvers/metric.ex

```diff
@@ -0,0 +1,10 @@
+defmodule GraphqlPracticeWeb.Resolvers.Metric do
+  alias GraphqlPractice.Metric
+
+  def get_count(_, %{request: request}, _) do
+    case Metric.get_count(request) do
+      nil -> {:ok, %{count: 0}}
+      count -> {:ok, %{count: count}}
+    end
+  end
```

Comment:
This functionality seems like it could be better put inside of the Metric module.